llama_index/embeddings/gradient.py

```diff
@@ -17,7 +17,7 @@
     "Represent this sentence for searching relevant passages:"
 )
 
-GRADIENT_MAX_BATCH_SIZE = 100
+GRADIENT_EMBED_BATCH_SIZE = 32_768
```

Comment:
is this a valid int number

Reply:
Yes! 

I'll patch it with, to make it easier to validate with mypy. 

```python
GRADIENT_EMBED_BATCH_SIZE: int = 32_768
```


```
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> type(32_768)
<class 'int'>
```